cli/types/tests/chainer-examples.ts

```diff
@@ -301,6 +301,19 @@ cy.wrap('foobar').should('not.have.string', 'baz')
 
 cy.wrap('foobar').should('not.include', 'baz')
 
+const mrObj = {
```

Comment:
so I like the test, can you 
- wrap it in a block similar to the typescript tests in other files for clarity
- what does the test below do WITHOUT the proposed `cy.invoke` type changes?

Reply:
Moved to new namespace inside `cypress-tests.ts`.

Without the proposed changes, none of the tests would pass except for the one with `bar.baz`.
```
ERROR: 326:3  expect  TypeScript@next expected type to be:
  Chainable<string>
got:
  Chainable<{ foo: (value: number, name?: string) => string; bar: { baz: (a: number) => number; }; }>

ERROR: 327:3  expect  TypeScript@next expected type to be:
Chainable<string>
got:
  Chainable<{ foo: (value: number, name?: string) => string; bar: { baz: (a: number) => number; }; }>

ERROR: 328:3  expect  TypeScript@next expected type to be:
  Chainable<number>
got:
  Chainable<{ baz: (a: number) => number; }>

ERROR: 329:1  expect  TypeScript@next: Expected an error on this line, but found none.
ERROR: 331:1  expect  TypeScript@next: Expected an error on this line, but found none.
ERROR: 332:1  expect  TypeScript@next: Expected an error on this line, but found none.
ERROR: 333:1  expect  TypeScript@next: Expected an error on this line, but found none.
```

Reply:
nice